src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +803,86 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    @property
+    def benchmark_metrics(self) -> pd.DataFrame:
```

Comment:
nit: can we instead do `get_benchmark_metrics`? Also, consider using `@lru_cache` annotation to simplify your logic